dbxio/delta/table_commands.py

```diff
@@ -200,6 +215,48 @@ def copy_into_table(
     client.sql(sql_copy_into_query).wait()
 
 
+def read_files_as_table(
+    client: 'DbxIOClient',
+    table: Table,
+    blob_path: str,
+    table_format: TableFormat,
+    abs_name: str,
+    abs_container_name: str,
+    include_files_pattern: bool = False,
+    replace: bool = False,
+    force_schema: bool = True,
+) -> None:
+    """
+    Copy data from blob storage as a table. All files that match the pattern *.{table_format} will be copied.
+    If force_schema == False it will use schemaHints instead of schema option
+    """
+    create_query = _create_table_query(table, replace, False, False)
+    options = {
+        'format': f"'{table_format.value.lower()}'",
+    }
+    if include_files_pattern:
+        options['fileNamePattern'] = f"'*.{table_format.value.lower()}'"
+    if table.schema:
+        sql_schema = f"'{table.schema.as_sql()}'"
+        if force_schema:
+            options['schema'] = sql_schema
+        else:
+            options['schemaHints'] = sql_schema
+    else:
+        options['mergeSchema'] = 'true'
+
+    options_query = ',\n'.join([f'{k} => {v}' for k, v in options.items()])
+    select_query = dedent(f"""
+    AS SELECT *
```

Comment:
function `read_files` sorts output columns in alphabetic order, so insert data can fail bc columns order mismatch

Reply:
It's better to replace `*` with `', '.join(table.schema.columns)`